api/dto/target.go

```diff
@@ -126,20 +127,25 @@ func TargetVerification(targets []string, ttl time.Duration, isRemote bool) []Tr
 			continue
 		}
 
-		functionsOfTarget.TreeOfProblems = checkExpression(expr, ttl, isRemote)
+		if expr.IsFunc() {
```

Comment:
i think this "if" is part of checkExpression

Reply:
I can move condition to checkExpression, and it would be like 
```go
	if !expression.IsFunc() {
		return nil
	}
```
at the start of function body. 
Do you think it would be better?